dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/FileOrchestrator.kt

```diff
@@ -32,6 +32,11 @@ internal interface FileOrchestrator {
      */
     fun getReadableFile(excludeFiles: Set<File>): File?
 
+    /**
+     * @return a List of all readable files ready for upload.
+     */
+    fun getAllReadableFiles(): List<File>
```

Comment:
This one is unnecessary too

Reply:
Actually I will need something similar, maybe I can just use `getReadableFile` and then call `getWritableFile` at the end to make sure I am sending also the one not returned by `getReadableFile`. I guess calling them in the right order:
`getWritableFile` and then calling `getReadableFile` until the batch is null will guarantee that everything is sent.